test/bigtable/table.js

```diff
@@ -0,0 +1,852 @@
+    });
+
+    it('should respect the gc rule option', function(done) {
+      var rule = {
+        a: 'a',
+        b: 'b'
+      };
+
+      table.request = function(g, reqOpts) {
+        assert.strictEqual(reqOpts.columnFamily.gcRule, rule);
+
+        var spy = FakeFamily.formatRule_;
+        assert.strictEqual(spy.callCount, 1);
+        assert.strictEqual(spy.getCall(0).args[0], rule);
+        done();
+      };
+
+      table.createFamily(COLUMN_ID, rule, assert.ifError);
+    });
+
+    it('should return an error to the callback', function(done) {
+      var error = new Error('err');
+      var response = {};
+
+      table.request = function(g, r, callback) {
+        callback(error, response);
+      };
+
+      table.createFamily(COLUMN_ID, function(err, family, apiResponse) {
+        assert.strictEqual(error, err);
+        assert.strictEqual(family, null);
+        assert.strictEqual(response, apiResponse);
+        done();
+      });
+    });
+
+    it('should return a Family object', function(done) {
+      var response = {
+        name: 'response-family-name'
+      };
+
+      table.request = function(g, r, callback) {
+        callback(null, response);
+      };
+
+      var familyFactory = table.family;
+
+      table.family = function(name) {
+        assert.strictEqual(name, response.name);
+        return familyFactory.apply(table, arguments);
+      };
+
+      table.createFamily(COLUMN_ID, function(err, family, apiResponse) {
+        assert.ifError(err);
+        assert(family instanceof Family);
+        assert.strictEqual(family.metadata, response);
+        assert.strictEqual(apiResponse, response);
+        done();
+      });
+    });
+  });
+
+  describe('deleteRows', function() {
+    it('should provide the proper request options', function(done) {
+      table.request = function(grpcOpts, reqOpts) {
+        assert.deepEqual(grpcOpts, {
+          service: 'BigtableTableService',
+          method: 'bulkDeleteRows'
+        });
+
+        assert.strictEqual(reqOpts.tableName, TABLE_NAME);
+        done();
+      };
+
+      table.deleteRows(assert.ifError);
+    });
+
+    it('should respect the row key prefix option', function(done) {
+      var options = {
+        prefix: 'a'
+      };
+
+      table.request = function(g, reqOpts) {
+        assert.strictEqual(reqOpts.rowKeyPrefix, options.prefix);
+
+        var spy = FakeMutation.convertToBytes;
+
+        assert.strictEqual(spy.callCount, 1);
+        assert.strictEqual(spy.getCall(0).args[0], options.prefix);
+        done();
+      };
+
+      table.deleteRows(options, assert.ifError);
+    });
+
+    it('should delete all data when no options are provided', function(done) {
+      table.request = function(g, reqOpts) {
+        assert.strictEqual(reqOpts.deleteAllDataFromTable, true);
+        done();
+      };
+
+      table.deleteRows(assert.ifError);
+    });
+
+    it('should return an error to the callback', function(done) {
```

Comment:
Kill this test. Just checking that `request` got the right `callback` is enough.